src/current/cockroachcloud/metrics-request-units.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Request Unit metrics
+summary: The Request Unit metrics let you monitor resource consumption.
+toc: true
+---
+
+{{site.data.alerts.callout_info}}
```

Comment:
I feel like I've seen this a lot of times.  is this text the same as the note at the top of the other pages?  if so I think this could be an include